app/scripts/services/Frame.coffee

```diff
@@ -66,6 +66,8 @@ angular.module('neo4jApp.services')
             @templateUrl = intr.templateUrl
             @startTime = (new Date).getTime() unless @startTime
             @pinTime = 0
+            @procedureNotFound = (error) ->
+              return true if error && error.search('ProcedureNotFound') != -1
```

Comment:
Details but to make it more coffee-scripty: `return yes if error and error.search('ProcedureNotFound') isnt -1`